manifests/test.pp

```diff
@@ -0,0 +1,15 @@
+# This should be called *before* any classes upon which you wish to enforce
```

Comment:
I forgot to move this class around as it's only used by the spec tests to test APL